index.js

```diff
@@ -344,10 +367,26 @@ function isLocal() {
 }
 
 function addDevModeMessageBox() {
-  $("body").prepend("<p class='alert'>DEVELOPER MODE</p>");
+  $("body").prepend("<p class='alert' id='devOn'><a class='devLink'>DEVELOPER MODE ON</a></p>");
 }
 
-if (devMode) {
+if (devModeCheck) {
   addDevModeMessageBox();
   testStonksify();
 }
+
+//Toggle Dev Mode on Local
+$(".devLink").on("click", function () {
```

Comment:
Can't the first thing just be devMode = !devMode; and then based on the checks on if this is true or not? That way the variable devMode is used as the conditional instead of class/id names.

Reply:
updated